BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -1820,6 +1835,13 @@ private RealSense2API.RS2StreamProfile GetProfileFromCapturedFrames(string chann
 
         unsafe public override RigidBodyTransformation GetExtrinsics(string channelFromName, string channelToName)
         {
+            // first check if intrincs for requested channel have been cached already
+            string extrinsicsKey = string.Format("{0}_{1}", channelFromName, channelToName);
```

Comment:
Just because it's fancy I would use string interpolation here: `$"{channelFromName}_{channelToName}"` :-)